profiles/rmfAudioCaptureAuxNotSupported.yaml

```diff
@@ -0,0 +1,5 @@
+rmfaudiocapture:
```

Comment:
we can add
extendedEnumsSupported: false as features list 
and remove the profile folder